app/grandchallenge/algorithms/forms.py

```diff
@@ -39,6 +134,7 @@ class Meta:
             "logo",
             "social_image",
             "public",
+            "inputs",
```

Comment:
Could we use `Select2MultipleWidget` for this?

Reply:
Or, maybe hide input selection until it is generally deployed?

Reply:
In that case we only allow it in the admin for now I assume?

Reply:
Yes, now that you have added the toggle.